src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -68,10 +68,12 @@ public function process(ContainerBuilder $container)
                     continue;
                 }
 
-                if (false !== strpos($definedTag, $tag) || levenshtein($tag, $definedTag) <= strlen($tag) / 3) {
-                    $candidates[] = $definedTag;
+                if (false !== strpos($definedTag, $tag) || $lev = levenshtein($tag, $definedTag) <= strlen($tag) / 3) {
+                    $candidates[$definedTag] = $lev;
```

Comment:
I'm not sure about this. `$lev` is the score ... and we need that for the two new lines added below to sort results from best to worst:

```php
asort($candidates);
$candidates = array_keys($candidates);
```